src/SamplesApp/UITests.Shared/Windows_UI_Notifications/ToastNotification.xaml.cs

```diff
@@ -0,0 +1,66 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Runtime.InteropServices.WindowsRuntime;
+using Uno.UI.Samples.Controls;
+using Windows.Data.Xml.Dom;
+using Windows.Foundation;
+using Windows.Foundation.Collections;
+using Windows.UI.Notifications;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Controls.Primitives;
+using Windows.UI.Xaml.Data;
+using Windows.UI.Xaml.Input;
+using Windows.UI.Xaml.Media;
+using Windows.UI.Xaml.Navigation;
+
+// The User Control item template is documented at https://go.microsoft.com/fwlink/?LinkId=234236
+
+namespace UITests.Windows_UI_Notifications
+{
+	[SampleControlInfo("Windows.UI.Notifications", "Toast")]
+	public sealed partial class ToastNotification : Page
+	{
+		public ToastNotification()
+		{
+			this.InitializeComponent();
+		}
+
+
+		private void sendToast_Click(object sender, RoutedEventArgs e)
+		{
+			string toastLine1 = uiToastLine1.Text;
+			string toastLine2 = uiToastLine2.Text;

```

Comment:
Let's add a sample option for the launch parameter.

Reply:
Field for entering Toast Launch added, but I don't know how to make sample to receive "toast tapped" event - as it means using App.OnActivated , and how to do it inside SamplesApp?

Reply:
@MartinZikmund will likely be able to help there.

Reply:
Marking as "resolved", as "That's fine for now, we'll do the launch parameters handling in another PR."